ParallelPrimitives/RadixSort.h

```diff
@@ -46,7 +46,7 @@ class RadixSort
 	/// @param kernelPath The kernel path.
 	/// @param includeDir The include directory.
 	/// @return The size of the temp buffer.
-	u32 configure( oroDevice device, const std::string& kernelPath = "", const std::string& includeDir = "" ) noexcept;
+	u32 configure( oroDevice device, OrochiUtils* oroutils, const std::string& kernelPath = "", const std::string& includeDir = "" ) noexcept;
```

Comment:
(same reason as above) Use reference instead of pointer since this cannot be null and yet we do not check for nullptr in the code.
```suggestion
	u32 configure( oroDevice device, OrochiUtils& oroutils, const std::string& kernelPath = "", const std::string& includeDir = "" ) noexcept;
```

Reply:
Also it misses the doxygen document update for `@param oroutils`